plugins/lookup/tss.py

```diff
@@ -82,6 +82,18 @@
                 | items2dict(key_name='slug',
                              value_name='itemValue'))['password']
             }}
+            
+- hosts: localhost
+  vars:
+      secret: "{{ lookup('community.general.tss',102,base_url='https://secretserver.domain.com/SecretServer/',username='user.name',password='password') }}"
```

Comment:
```suggestion
      secret: "{{ lookup('community.general.tss', 102, base_url='https://secretserver.domain.com/SecretServer/', username='user.name', password='password') }}"
```
Or even better, split it up into multiple lines to make it possible to read it without having to scroll on medium-width browser windows.

Reply:
Thanks for the suggestion, I've committed that now

Reply:
You know what, your other suggestion is better since that line is particularly monstrous. I will split to different lines